debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresReplicationConnection.java

```diff
@@ -203,17 +203,16 @@ else if (shouldCreateSlot || !slotInfo.hasValidFlushedLsn()) {
     }
 
     private boolean useTemporarySlot() throws SQLException {
-        return dropSlotOnClose && isPg10OrGreater();
+        return dropSlotOnClose && isPg94OrGreater();
```

Comment:
PostgreSQL did not add _Temporary Replication Slots_ until PG10, so I believe we need to keep the `isPg10OrGreater` method and revert this one line back to its original behavior.  @cscetbon can you make that change for me and run the test suite again to see if it fixes the problems on 9.6?

Reply:
@Naros I sure can. And there is nothing else to do when it does not use temporary slots ?

Reply:
Nope, that was the only issue that I saw.